src/main/groovy/liquibase/harness/util/JSONUtils.groovy

```diff
@@ -70,7 +74,10 @@ class JSONUtils {
             for (int j = 0; j < jsonArrayToCompare.length(); j++) {
                 def jsonObjectRight = new JSONObject(jsonArray.get(i).toString())
                 def jsonObjectLeft = new JSONObject(jsonArrayToCompare.get(j).toString())
-                def result = JSONCompare.compareJSON(jsonObjectLeft, jsonObjectRight, JSONCompareMode.NON_EXTENSIBLE)
```

Comment:
@XDelphiGrl I added CustomComparator to be able to leave NON_EXTENSIBLE but match by values by regexp, that should make our checks more precise

Reply:
@KushnirykOleh, thank you!!! I knew there had to be a better way to handle this particular case. Thank you for making my overly-broad code change much more precise!